openclaw_hooks/nima-recall-live/lazy_recall_legacy_v1.py

```
@@ -0,0 +1,468 @@
+    if len(sys.argv) < 2:
+        print("Usage: lazy_recall.py <query> [affect_json]", file=sys.stderr)
+        sys.exit(1)
+    
+    query = sys.argv[1]
+    current_affect = None
+    
+    # Parse current affect if provided
+    if len(sys.argv) > 2:
+        try:
+            current_affect = json.loads(sys.argv[2])
+        except:
```

Ruff (0.15.0)

[error] Do not use bare `except`

(E722)